src/utils/time-utils.ts

```diff
@@ -171,4 +171,25 @@ const countWorkingWeekDaysInRange = (startWeekIndex: number, endWeekIndex: numbe
     if (workingWeek[i-1]) takenWorkingDays+=1;
   }
   return takenWorkingDays;
+}
+
+/**
+ * Get sprint start date
+ *
+ * @param date string date 
+ */
+export const getSprintStart = (date: string) => {
+  const weekIndex = DateTime.fromISO(date).localWeekNumber;
+  const weekDay = DateTime.fromISO(date).weekday;
+  const days = (weekIndex%2 === 1 ? 0 : 7) + weekDay;
+  return DateTime.fromISO(date).minus({days : days - 1});
+}
+
+/**
+ * Get sprint end date
+ *
+ * @param date string date 
+ */
+export const getSprintEnd = (date: string) => {
+  return getSprintStart(date).plus({days : 14});
```

Comment:
We might want to change this e.g. currently the endDate is the same Monday as the start date of the next sprint, perhaps it should be the Friday before e.g. days: 11?
What do you think is best?